services/static-webserver/client/source/class/osparc/data/model/NodeProgressSequence.js

```diff
@@ -181,96 +212,109 @@ qx.Class.define("osparc.data.model.NodeProgressSequence", {
         case "SERVICE_IMAGES_PULLING":
           this.setImagesPulling(progress);
           break;
+        case "SERVICE_INPUTS_PULLING":
+          this.setInputsPulling(progress);
+          break;
       }
     },
 
     __initLayout: function() {
-      this.__sequenceLoadingPage = new qx.ui.container.Composite(new qx.ui.layout.VBox(8)).set({
-        maxWidth: 280,
-        minHeight: 250
-      });
+      this.__mainLoadingPage = new qx.ui.container.Composite(new qx.ui.layout.VBox(8));
 
-      const scalingTitle = this.__clusterUpScalingTitle = this.self().createTitleAtom(qx.locale.Manager.tr("Increasing system capacity..."));
-      this.__sequenceLoadingPage.add(scalingTitle);
 
-      const scalingSubtitle = this.__clusterUpScalingSubtitle = new qx.ui.basic.Label(qx.locale.Manager.tr("This step can take up to 3 minutes"));
-      scalingSubtitle.exclude();
-      this.__sequenceLoadingPage.add(scalingSubtitle);
+      const sequenceLoadingPage = new qx.ui.container.Composite(new qx.ui.layout.VBox(9)).set({
+        backgroundColor: "window-popup-background"
+      });
 
-      const clusterUpScalingPBar = this.__clusterUpScalingPBar = this.self().createProgressBar();
-      this.__sequenceLoadingPage.add(clusterUpScalingPBar);
+      const progressTitle = new qx.ui.basic.Label(qx.locale.Manager.tr("LOADING...")).set({
+        font: "text-12",
+        alignX: "center",
+        alignY: "middle",
+        margin: 10
+      });
+      const defaultPBar = this.__defaultProgressBar = this.self().createProgressBar(6);
+      sequenceLoadingPage.add(progressTitle);
+      sequenceLoadingPage.add(defaultPBar);
+
+      const defaultProgressSubtitle = this.__defaultProgressSubtitle = new qx.ui.basic.Atom().set({
+        label: qx.locale.Manager.tr("Please be patient, this process can take up to 5 minutes..."),
```

Comment:
If you show the message after 50", I would say that it can take up to 3'

Reply:
But then the user is expecting it to take aprox. 2 mins as 1 has passed?